trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..5]
+  end
+end
```

Comment:
- [ ] Here please use the methods itself instead of calling super
- [ ] Also you did not implement this requirement 
![image](https://github.com/sanieni6/OOP-school-library/assets/62705947/396c3ba1-b60b-4a95-8706-43e43d2008b5)